friends/src/App.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import './App.css';
+
+import FriendsList from './components/FriendsList';
+import FriendForm from './components/FriendForm';
+import UpdateFriendForm from './components/UpdateFriendForm';
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: [],
+      friend: {
+        name: '',
+        age: '',
+        email: ''
+      },
+      updating: false
+    };
+  }
+
+  componentDidMount() {
+    axios.get('http://localhost:5000/friends')
+      .then(res => {
+        this.setState({
+          friends: res.data
+        })
+      })
+      .catch(err => console.log(err));
+  }
+
+  handleChanges = e => {
+    this.setState({
+      friend: {
+        ...this.state.friend,
+        [e.target.name]: e.target.value
+      }
+    });
+  }
+
+  addFriend = friend => {
```

Comment:
Add a console.log(friend) here to see what you are passing into the method when it is invoked